src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -283,6 +283,79 @@
                     </Setter.Value>
                 </Setter>
             </Style>
+            <Style x:Name="GraphViewToggleButtonStyle"
```

Comment:
I liked the old style better, the toggle button is confusing imo, I am not sure when it is on/off base solely based on the font icon. It was clearer when the background was blue like the trace toggle button on the top of row.

Reply:
The current design is to only fill in the circle with no background color change.

Reply:
In that case I think we should update the tool tip to let the user know which state the button will go to when clicked, it isn't clear which icon is on/off to me